### PR TITLE
Improving A11y across pages

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Puts us in green for the majority of the pages on google lighthouse for A11y

Fixes # (issue)

## Type of change

- [ X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

No Tests needed.

## Checklist

- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by zomars — CHANGES_REQUESTED** [review]
Nice work. Minor nitpick. Are we adding/removing any dependencies? If we're not let's skip the yarn.lock file from this PR please 🙏

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
once merge conflcit solved it'll auto merge @sean-brydon

**Comment by PeerRich** [comment]
image
can be merged after merge conflicts were resolved 🙏 @sean-brydon

**Review by zomars — APPROVED** [review]

**Pull request merged**
